Formula/wireguard-tools.rb

```diff
@@ -0,0 +1,19 @@
+class WireguardTools < Formula
+  desc "Tools for the WireGuard secure network tunnel"
+  homepage "https://www.wireguard.io/"
+  # Though this tools package is from an experimental tag, these tools themselves are
```

Comment:
At the risk of being annoying _(Which I specialise in)_, can we trim this a little bit? Something like:
```ruby
# Despite the experimental tag the tools themselves are stable.
# Please only update version when the tools have been modified/updated.
```